test/PlaneWaveBasis.jl

```diff
@@ -58,7 +58,7 @@ end
     test_pw_cutoffs(silicon, 4.0, [11, 13, 11])
 end
 
-@testitem "PlaneWaveBasis: Check cubic basis and cubic index" setup=[TestCases] begin
+@testitem "PlaneWaveBasis: Check cubic basis and cubic index" setup=[TestCases] tags=[:dont_test_mpi] begin
```

Comment:
Just to check: Make sure that at least one test tests the code path with k-point duplication, because missing errors in this from refactors can be super annoying.

Reply:
Also in a follow-up we should really invert from `:dont_test_mpi` to `:mpi` on all mpi-enabled tests.

Reply:
I don't mind doing this as part of this PR

Reply:
This was not a comment specifically to this test, but more a general remark.